.github/workflows/ci_submodule.yml

```diff
@@ -0,0 +1,72 @@
+name: CI for libsc check
+on:
+  pull_request:
+
+jobs:
+
+  linux-sub:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    env:
+      TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NODE_ID: ${{ github.event.pull_request.node_id }}
+      INPUT_PATH: sc
+    steps:
+    - name: Checkout source code for base
+      uses: actions/checkout@v4
+      with:
+        submodules: true
```

Comment:
Would this work with submodules false and just checking the sc hash relative to each p4est subdirectory?

Reply:
Thanks for the idea. It is fixed in the recent update.